### PR TITLE
feat(lpms-server): server bootstrap

### PR DESCRIPTION
**Pull request by Galeaf11**
added BootstrapService.ts
added WalletService.ts and WalletController.ts
added default user process

### CODE REVIEW

File: packages/lpms-server/.env.example [hunk 1]
```diff
@@ -3,3 +3,4 @@ CLIENT_URL=http://localhost:3000
 APP_PORT=5000
 APP_ACCESS_TOKEN_KEY=
 APP_REFRESH_TOKEN_KEY=
+APP_WALLET_KEY=
```
Comment (mfw78) [inline packages/lpms-server/.env.example:6]: Recommend to rename the key to `APP_WALLET_PASSPHRASE` to keep with standard phraseology.

File: packages/lpms-server/src/services/BootstrapService.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      throw e;
+    }
+  }
+
+  private async configure(): Promise<void> {
+    const userService = new UserService();
+
+    await userService.createUser(
+      defaultManagerLogin,
+      'winwin',
```
Comment (mfw78) [inline packages/lpms-server/src/services/BootstrapService.ts:42]: Abstract constants to the header of the file, ie. `const DEFAULT_PASSWORD`

**Review by mfw78 — CHANGES_REQUESTED** [review]

File: packages/lpms-server/.env.example (hunk 1 above)
Comment (kostysh) [reply packages/lpms-server/.env.example]: PR is edited, APP_WALLET_KEY renamed to APP_WALLET_PASSPHRASE in the `./.env.example`

**Review by kostysh — APPROVED** [review]
ready to be merged

File: packages/lpms-server/src/services/BootstrapService.ts (hunk 2 above)
Comment (kostysh) [reply packages/lpms-server/src/services/BootstrapService.ts]: done

**Pull request merged**